doc/source/_static/numpy.css

```diff
@@ -71,4 +72,33 @@ div.admonition-legacy>.admonition-title::after {
 
 div.admonition-legacy>.admonition-title {
   background-color: var(--pst-color-warning-bg);
-}
\ No newline at end of file
+}
+
+/* Buttons for JupyterLite-enabled interactive examples */
+
+.try_examples_button {
+  color: white;
+  background-color: #0054a6;
```

Comment:
we should ideally be using colors from the PyData Sphinx theme color scheme, so we get any accessibility/dark mode/usability improvements as they come.

Reply:
Thanks – I'll try to experiment with the colours based on their scheme and update this

Reply:
Can we just delete this whole file? I can't remember if there were any blockers for doing so. The pydata-sphinx-theme defaults should be better than what we have here.

Reply:
Using the PyData Theme defaults should be good, I will take a look at it (in another PR or here? Fine with either). However, we'll have to keep the changes for these buttons in specific, because they aren't styled particularly well by default:

<img width="669" alt="image" src="https://github.com/numpy/numpy/assets/74401230/92f94a9d-c42b-48b8-844f-1cc8aaaab365">

Reply:
In 0c7908f0485b72c2a1093286dc2cc1c92e64cb82, I decided to use `--pst-color-info` here, it seems to be the best suited for the examples and is easy on the eyes, besides `--pst-color-primary`.

Reply:
Resolving this conversation for now, please let me know if there's something I can help out with conforming to other accessibility guidelines.